### DISCUSSION TITLE
Stable `goto_word` labels

### DISCUSSION THREAD

**Discussion by miquella** [helix-editor/helix#10646] [Ideas]
Using `goto_word` today, I was caught off-guard by something and thought I would bring it up here!

In trying to jump to a particular place in the document, I typed `g` `w` (`goto_word`), which displayed something like this:
![image](https://github.com/helix-editor/helix/assets/2895155/77c05dd3-038c-450b-9ce3-85d12c8edc80)

"Perfect, `du` is where I'm trying to go!" I thought. Only to realize that I mistyped it: 🤦
![image](https://github.com/helix-editor/helix/assets/2895155/015a939b-7c95-4586-9be7-f0f37b4aac5c)

"No problem, I'll just try again." So I typed `g` `w` again, followed by `du` … but all of the labels had shifted:
![image](https://github.com/helix-editor/helix/assets/2895155/9e24c5ff-ec69-4af8-b8e7-699be0acf77c)

Which lead me to a question: if the viewport and text haven't changed, just the cursor position, would it make sense for the `goto_word` labels to be stable? That way you wouldn't having to stop to figure out what the new label is each time.

**Comment by the-mikedavis** [comment]
I'm not sure I have a strong preference between this and the current behavior. The current behavior puts characters that are earlier in the jump label alphabet closer to your cursor though so it can be easier to make smaller jumps, if you customize your alphabet.
